task.go

```diff
@@ -433,6 +435,11 @@ func (t *task) Checkpoint(ctx context.Context, opts ...CheckpointTaskOpts) (Imag
 	if err := t.checkpointTask(ctx, &index, request); err != nil {
 		return nil, err
 	}
+	// if checkpoint image path passed, jump checkpoint image
+	if isCheckpointPathExist(cr.Runtime.Name, i.Options) {
+		return nil, nil
```

Comment:
I think we can return error if the `isCheckpointPathExist` return true. because the caller will use the nil Image

Reply:
It is normally exit here, error is not suit, or a empty image ? I am not sure

Reply:
overall this looks good, i'm only nervous about this line to cause panics to the caller.  We should maybe update the comment of the method to note that Image will be nil when checkpointed to to a directory.

Reply:
I return an empty image instead of nil image, and I test that `empty image` not be panic , is it right? I add comment for the `Checkpoint` function, @crosbymichael 
( the btrfs on my machine get crashed yesterday night, I spend whole morning restore my data, so it late to update)